pages/story/[uid].js

```diff
@@ -0,0 +1,37 @@
+import { client } from "../../prismic-configuration";
+import { RichText } from "prismic-reactjs";
+import Prismic from "prismic-javascript";
+import SliceZone from "../../components/slices/SliceZone";
+
+export default function Story({ data }) {
+  return (
+    <article>
+      <header>{RichText.asText(data.title)}</header>
+      <SliceZone sliceZone={data.body} />
+    </article>
+  );
+}
```

Comment:
I also recommend using PropTypes

Reply:
Discussed offline: looking to add PropTypes once we've stabilized on prismic data.